.eslintrc.js

```diff
@@ -37,6 +38,10 @@ module.exports = {
   },
 
   overrides: [
+    {
+      files: ['test/js/**/*.{ts,js}'],
```

Comment:
Should this be
```suggestion
      files: ['tests/js/**/*.{ts,js}'],
```

or is this for another directory I'm not seeing?